code/mod_table/mod_table.h

```diff
@@ -57,6 +57,7 @@ extern float Generic_pain_flash_factor;
 extern float Shield_pain_flash_factor;
 extern float Emp_pain_flash_factor;
 extern std::tuple<float, float, float> Emp_pain_flash_color;
+extern gameversion::version Targeted_version;
```

Comment:
`Targeted_version` is scoped to mod_table.cpp, and the standard use is to call `mod_supports_version`.  I would favor adding an overload of `mod_supports_version` that works on `gameversion::version` rather than exposing the variable.